public/js/angular/controllers.js

```diff
@@ -254,32 +246,32 @@ angular
       $scope.weblit = weblit;
 
       $scope.wlcPoints = CONFIG.wlcPoints;
-
     }
   ])
   .controller('competencyMediaGenController', ['$scope', '$sce',
     function ($scope, $sce) {
       function safeUrl(media) {
-        switch (media.type) {
-        case 'youtube':
-          return $sce.trustAsResourceUrl('https://www.youtube-nocookie.com/embed/' + media.content + '?enablejsapi=1&amp;wmode=transparent&amp;autohide=1&amp;autoplay=0&amp;rel=0&amp;fs=1&amp;hd=1&amp;rel=0&amp;showinfo=0&amp;start=&amp;theme=dark');
-        case 'vimeo':
-          return $sce.trustAsResourceUrl('https://player.vimeo.com/video/' + media.content + '?title=0&amp;byline=0&amp;portrait=0&amp;color=eb6933');
-        case 'ted':
-          return $sce.trustAsResourceUrl('https://embed-ssl.ted.com/talks/' + media.content + '.html');
-        case 'popcorn':
-          return $sce.trustAsResourceUrl(media.content);
-        default:
+        var sources = {
+          // jscs:disable maximumLineLength
```

Comment:
Gooood call.

Was 2 more long strings that I was missing because of my oversight here.